.github/workflows/oxfmt-ci.yml

```
@@ -0,0 +1,214 @@
+name: oxfmt ecosystem ci
+
+on:
+  schedule:
+    - cron: "0 */6 * * *" # Every 6 hours
```

We need to load balance these heavy crons (shift to some other minutes) otherwise jobs will get queued for a long time.

---

oxlint-ci runs on: 0:00, 3:00, 6:00, 9:00, 12:00, 15:00, 18:00, 21:00

```suggestion
    - cron: "30 1,7,13,19 * * *" # Every 6 hours, gaps in the oxlint-ci
```

By this, oxfmt-ci runs on: 1:30, 7:30, 13:30, 19:30